qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -104,13 +104,16 @@
         - with_iommu:
             only virtio_net
             only nic_virtio
-            netdev_extra_params_hotplug_nic1 = ",vhost=on,vhostforce=on"
+            netdev_extra_params_hotplug_nic1 = ",vhost=on"
             nic_extra_params_hotplug_nic1 = ",disable-legacy=on,disable-modern=off,iommu_platform=on,ats=on"
             x86_64:
                 only q35
                 no WinXP WinVista Win7 Win8 Win8.1 Win2003
                 no Win2008 Win2008..r2 Win2012 Win2012..r2
                 machine_type_extra_params = "kernel-irqchip=split"
+                virtio_dev_iommu_platform = on
+                virtio_dev_ats = on
+                iommu_caching_mode = on
```

Comment:
Same with above.

Reply:
Hello @yanan-fu Thanks for your help. I will change it.